pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -62,11 +63,13 @@ func NewCommand(f client.Factory) *cobra.Command {
 		Long: `Uninstall Velero along with the CRDs and clusterrolebinding.
 
 The '--namespace' flag can be used to specify the namespace where velero is installed (default: velero).
-Use '--wait' to wait for the Velero uninstall to be ready before proceeding.
 Use '--force' to skip the prompt confirming if you want to uninstall Velero.
 		`,
 		Example: ` # velero uninstall --namespace staging`,
 		Run: func(c *cobra.Command, args []string) {
+			if o.wait {
+				fmt.Println("Warning: the \"--wait\" option is deprecated and will be removed in the future release. The uninstall command always waits the uninstall to be ready currently.")
```

Comment:
will be removed in the future release -> will be removed in a future release

waits the uninstall to be ready currently. -> waits for the uninstall to complete